bedrock/firefox/templates/firefox/whatsnew/index.zh-TW.html

```diff
@@ -26,33 +26,41 @@
 {% endblock %}
 
 {% block content %}
+<header class="main-header">
```

Comment:
Same suggestion as the PR for `/id/`, we could move this header (and it's CSS) to a shared component?